src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder report = new StringBuilder();
```

Comment:
There is should be 3 methods - readFile, createReport and writeToFile.

Reply:
First, don't need to declare variable that you don't actually use in this method 
Second, readCalculateFile has one task - read file, create report - is another task 
Look at common mistake file [Divide program logic](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#divide-program-logic)